docs/local.md

```diff
@@ -65,7 +91,7 @@ Now get the ethereum contract address that Alice deployed to. This can be pulled
 the file ..., or if you have `jq` installed (available via `sudo apt install jq`), you can set a
 variable like this:
 ```bash
-CONTRACT_ADDR=$(jq -r .ContractAddress /tmp/xmrtaker-*/contract-address.json)
+CONTRACT_ADDR=$(jq -r .ContractAddress "${TMPDIR-/tmp}"/xmrtaker-*/contract-address.json)
```

Comment:
`"${TMPDIR-/tmp}"` - what is this syntax? is it just the os tempdir?

Reply:
`TMPDIR` is always defined on macOS, and it is something that `mktemp` and the Go libraries will use if it is defined on Linux (but it is not typically defined on Linux systems).  The bash syntax here uses the variable before the `-` if the variable is defined and non-empty, or the value after the `-` if `TMPDIR`is not defined.  Effectively, `TMPDIR` will get used on macOS while `/tmp` will be used on typical Linux systems.